extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -322,11 +322,20 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 	}
 
 	public async searchJupyterBooks(treeItem?: BookTreeItem): Promise<void> {
-		if (this.currentBook && this.currentBook.bookPath) {
-			let folderToSearch = this.currentBook.bookPath;
-			if (treeItem && treeItem.uri) {
-				folderToSearch = path.join(folderToSearch, Content, path.dirname(treeItem.uri));
+		let folderToSearch: string;
+		if (treeItem && treeItem.book.type !== BookTreeItemType.Notebook) {
+			if (treeItem.url) {
+				folderToSearch = path.join(treeItem.root, ContentFolder, path.dirname(treeItem.url));
+			} else {
+				folderToSearch = path.join(treeItem.root, ContentFolder);
 			}
+		} else if (this.currentBook && !this.currentBook.isNotebook && this.currentBook.bookPath) {
+			folderToSearch = path.join(this.currentBook.bookPath, ContentFolder);
```

Comment:
@MaddyDev Is appending Content fine here? Previously it was just bookPath.

Reply:
as long as the currentBook isn't notebook, the bookPath is the root folder of the book and appending contentFolder is fine.